docs/framework-introscope_agent.md

```diff
@@ -1,5 +1,5 @@
-# Introscope Agent Framework
-The Introscope Agent Framework causes an application to be automatically configured to work with a bound [Introscope service][].  **Note:** This framework is disabled by default.
+# CA APM Framework
```

Comment:
We have discussed this internally again, and Introscope hasn't been in use for the past few years, and customers know us as CA APM. Is there a way to change the name of the framework in the user facing documentation, without changing anything else (Ruby files, artifacts, etc.)? We would like to make it easy for our customers.